app/views/citizens/additional_accounts/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :navigation do %>
+  <%#= link_to 'Back', path-to-be-determined, class: 'govuk-back-link', id: 'back' %>
+<% end %>
+
+  <%= form_tag(citizens_additional_account_path(id: :update), method: :patch) do %>
+
+    <%= govuk_form_group(
+          show_error_if: @error,
+          hint_id: :has_offline_accounts_hint,
+          input: :has_offline_accounts
+        ) do %>
+      <%= govuk_fieldset_header(
+            'Do you have online access to your accounts with other banks?',
+            padding_below: 6
+          ) %>
+      <%= govuk_hint(
+            'If you do not, you will need to provide paper evidence for these accounts.',
+            id: :has_offline_accounts_hint
+          ) %>
+      <%= govuk_error_message(@error) %>
+      <%= govuk_radio_inputs :has_offline_accounts, yes: 'Yes', no: 'No' %>
+    <% end %>
+
+    <%= govuk_submit_button 'Continue' %>
+
+  <% end %>
+
+  </div>
```

Comment:
[Ditto](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/64/files#r232273315)

Reply:
Ditto